src/core/main.c

```diff
@@ -385,7 +385,7 @@ sigterm_handler (int signum)
 }
 
 static gboolean
-on_sigterm (void)
+on_sigterm (GIOChannel *source, GIOCondition condition, gpointer user_data)
```

Comment:
Seems this is not exactly true in this file, but this is not style used in this code base. Each parameter should on new line.

Reply:
ok

Reply:
see [GIOFunc](https://developer.gnome.org/glib/stable/glib-IO-Channels.html#GIOFunc) function prototype